docs/v3.x/concepts/middlewares.md

```diff
@@ -182,6 +182,7 @@ The session doesn't work with `mongo` as a client. The package that we should us
 
 - [`gzip`](https://en.wikipedia.org/wiki/Gzip)
   - `enabled` (boolean): Enable or not GZIP response compression.
+  - `enableBrotli` (boolean): Enable Brotli encoding. Default value: `false`.
```

Comment:
Allowing to pass any param,you can just direct tot he middleware libe directly ;)

Reply:
Done